views/signUp.hbs

```diff
@@ -0,0 +1,32 @@
+{{> header}}
+<section class="section-signup">
+    <div class="row">
+        <div class="signup-text">
+            <p>Signup</p>
```

Comment:
Define common styling for this heading and description, as this is being used in multiple pages.
Should be H1